tests/test_jwt.py

```diff
@@ -196,6 +198,7 @@ def setUp(self):
 
 
 class PEMCryptTestsOpenSSL(CryptTests):
+
```

Comment:
Just consistency with `CryptTests`. I thought the extra line was a requirement for the linter, but after some experimenting it doesn't seem to be.